src/serverdlg.cpp

```diff
@@ -136,7 +136,7 @@ CServerDlg::CServerDlg ( CServer* pNServP, CServerSettings* pNSetP, const bool b
 #ifdef _WIN32
         + tr ( "Recording has been switched off by the UI checkbox." )
 #else
-        + tr ( "Recording has been switched off, either by the UI checkbox or SIGUSR2 being received." )
+        + tr ( "Recording has been switched off, by UI checkbox, SIGUSR2 or JSON-RPC." )
```

Comment:
JSON-RPC is available on Windows, too.  It's only SIGUSR2 that doesn't apply.

Reply:
Thanks. Fixed.